modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/export_portlet.jsp

```diff
@@ -463,7 +467,7 @@ PortletURL portletURL = PortletURLBuilder.createRenderURL(
 								/>
 							</c:if>
 						</div>
-					</div>
+					</div
```

Comment:
You lost `>` here.

Reply:
@tototrinh  sorry my mistake. I fixed it

Reply:
@thabet91 I can't see any change here. Please double-check. Thanks.

Reply:
@tototrinh sorry again there was an issue with my rebase I had to fix it. can you please check now.